rubin_sim/maf/metrics/technical_metrics.py

```diff
@@ -164,7 +164,7 @@ def run(self, data_slice, slice_point=None):
         # Sort on time, to be sure we've got filter (or other col) changes in the right order.
         idxs = np.argsort(data_slice[self.time_col])
         changes = data_slice[self.change_col][idxs][:-1] != data_slice[self.change_col][idxs][1:]
-        condition = np.where(changes == True)[0]
+        condition = np.where(changes is True)[0]
```

Comment:
ruff is a little over-eager about these replacements. The changes actually make the metrics fail.